test/es-module/test-esm-loader-modulemap.js

```diff
@@ -43,8 +45,8 @@ common.expectsError(
   {
     code: 'ERR_INVALID_ARG_TYPE',
     type: TypeError,
-    message: 'The "job" argument must be of type ModuleJob. ' +
-             'Received type string'
+    message: 'The "job" argument must be an instance of ModuleJob. ' +
+             "Received type string ('notamodulejob')"
```

Comment:
Micro-nit, but in messages like this one, I think it's more idiomatic for strings to be in double-quotes. Of course, both are fine in JavaScript so feel free to ignore this. People will know what is meant. (Only noticed it myself because `job` on the line above is in double quotes, but that's not even a string, it's an identifier, so 
¯\\_(ツ)_/¯ but I did have to stop and wonder why one was in double-quotes and another in single-quotes.)

Reply:
I rely upon `.inspect()` in this case and it's not possible to configure the quotation in that case. Therefore I'd stick to this for now (even though it would be possible to special handle strings).